src/mouse.js

```diff
@@ -52,49 +53,68 @@ var mouseEvents = {
     e.pointerId = this.POINTER_ID;
     e.isPrimary = true;
     e.pointerType = this.POINTER_TYPE;
-    if (!HAS_BUTTONS) {
-      e.buttons = WHICH_TO_BUTTONS[e.which] || 0;
-    }
     return e;
   },
+  prepareButtonsForMove: function(e, inEvent) {
+    var p = pointermap.get(this.POINTER_ID);
+    e.buttons = p ? p.buttons : 0;
+    inEvent.buttons = e.buttons
+  },
   mousedown: function(inEvent) {
     if (!this.isEventSimulatedFromTouch(inEvent)) {
-      var p = pointermap.has(this.POINTER_ID);
+      var p = pointermap.get(this.POINTER_ID);
       // TODO(dfreedman) workaround for some elements not sending mouseup
       // http://crbug/149091
-      if (p) {
-        this.cancel(inEvent);
-      }
       var e = this.prepareEvent(inEvent);
+      if (!HAS_BUTTONS) {
+        e.buttons = BUTTON_TO_BUTTONS[e.button];
+        if (p) e.buttons |= p.buttons;
+        inEvent.buttons = e.buttons
+      }
       pointermap.set(this.POINTER_ID, inEvent);
-      dispatcher.down(e);
+      if(!p)
+        dispatcher.down(e);
+      else
+        dispatcher.move(e);
     }
   },
   mousemove: function(inEvent) {
     if (!this.isEventSimulatedFromTouch(inEvent)) {
       var e = this.prepareEvent(inEvent);
+      if (!HAS_BUTTONS) this.prepareButtonsForMove(e, inEvent);
       dispatcher.move(e);
     }
   },
   mouseup: function(inEvent) {
     if (!this.isEventSimulatedFromTouch(inEvent)) {
       var p = pointermap.get(this.POINTER_ID);
-      if (p && p.button === inEvent.button) {
-        var e = this.prepareEvent(inEvent);
-        dispatcher.up(e);
+      if (!p) return;
```

Comment:
Thank you for the hint - there is a mistake in the PR here:

This condition occurs if an uncaptured `mousedown` event from outside the website area started a mouse movement that ends with a `mouseup` on an element we observe.

I looked deeper into this issue today:
  - We can get a `mouseup` event for which we do not yet have an entry in the `pointermap`.
  - We should fire a `pointerup` event in this case as well (for consistency with the other pointer events, but also because the native implementation of e.g. IE does so)
  - If we have `inEvent.buttons` this is not a problem - but if we don't, we can't reconstruct the `buttons` value from the history of mouse events (because there is none...)

Therefore: The condition should definitely be removed (I'll do so in a second), but I'm not sure how to deal with browsers that do not have `buttons` and need the polyfill.